test/helpers/privateMocksSpec.js

```diff
@@ -9,7 +9,7 @@ describe('private mocks', function() {
         spyOn(window, 'it');
 
         they('should do stuff with $prop', ['a', 'b', 'c']);
-        expect(window.it.calls.length).toEqual(3);
```

Comment:
I still think the new way is better and the way that is present in Jasmine documentation